components/core/src/clp_s/clp-s.cpp

```diff
@@ -113,9 +123,15 @@ bool compress(CommandLineArguments const& command_line_arguments) {
     }
 
     clp_s::JsonParser parser(option);
-    if (false == parser.parse()) {
-        SPDLOG_ERROR("Encountered error while parsing input");
-        return false;
+    if ("IR" == file_type) {
+        // Functionality Coming in later PR
+        //  -->Call new parsing function in Json Parser to parse IRv2 to archive
+        //  -->Check for error from parsing function
```

Comment:
Should log an error and return false so that the error gets reported explicitly to the user.

Reply:
I was going to follow how the code was already handling false return value from parser.parse().

Reply:
Sorry I meant in general for this code block we don't do anything and this function returns true so it hides the fact that this feature isn't supported yet, and makes it look like ingestion was succesful.

To make it explicit that the feature isn't available in this PR we should add a log message in this branch that says as much, and return false.